prime-router/docs/api/delivery.yml

```diff
@@ -41,6 +41,29 @@ paths:
           description: The search request could not be parsed
         '404':
           description: The receiver could not be found
+  /waters/report/{reportId}/delivery/etorMetadata:
+    get:
+      summary: Retrieves ETOR metadata for a report
+      description: Allows fetching of ETOR intermediary metadata by report ID
```

Comment:
The summary and description for this and `/waters/report/{reportId}/history/etorMetadata` are the same, can we be more detailed in each case?

Reply:
Fixed.